remotehelper/runner.go

```diff
@@ -241,9 +241,8 @@ func (r *Runner) Run(ctx context.Context, remoteName string, remoteUrl string) e
 		// Connect can be used for upload / receive pack
 		// case "connect":
 		// 	r.respond("fallback\n")
-		case "": // empty line separates commands, return new line to end command
-			r.respond("\n")
-			break
+		case "": // command stream terminated, return out
```

Comment:
s/out/nil/ to match code?

Reply:
yeah my intention was the latter because the `for {` will keep going unless you "exit out" here. The 1 cp]ode was caused by having the break here, which resulted in an extra `for` iteration which produced an EOF